addons/l10n_es_edi_sii/models/account_tax.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class SIIAccountTaxMixin(models.AbstractModel):
+    _name = 'l10n_es.sii.account.tax.mixin'
+    _description = 'SII Fields'
+
+    l10n_es_exempt_reason = fields.Selection(
+        selection=[
+            ('E1', 'Art. 20'),
+            ('E2', 'Art. 21'),
+            ('E3', 'Art. 22'),
+            ('E4', 'Art. 23 y 24'),
+            ('E5', 'Art. 25'),
+            ('E6', 'Otros'),
+        ],
+        string="Exempt Reason (Spain)",
+    )
+    l10n_es_type = fields.Selection(
+        selection=[
+            ('exento', 'Exento'),
+            ('sujeto', 'Sujeto'),
+            ('sujeto_agricultura', 'Sujeto Agricultura'),
+            ('sujeto_isp', 'Sujeto ISP'),
+            ('no_sujeto', 'No Sujeto'),
+            ('no_sujeto_loc', 'No Sujeto por reglas de Localization'),
+            ('no_deducible', 'No Deducible'),
+            ('retencion', 'Retencion'),
+            ('recargo', 'Recargo de Equivalencia'),
+            ('ignore', 'Ignore even the base amount'),
+        ],
+        string="Tax Type (Spain)", default='sujeto'
+    )
+    l10n_es_bien_inversion = fields.Boolean('Bien de Inversion', default=False)
+
+
+class AccountTax(models.Model):
+    _inherit = ['account.tax', 'l10n_es.sii.account.tax.mixin']
+    _name = 'account.tax'
+
+
+class AccountTaxTemplate(models.Model):
+    _inherit = ['account.tax.template', 'l10n_es.sii.account.tax.mixin']
+    _name = 'account.tax.template'
+
+    def _get_tax_vals(self, company, tax_template_to_tax):
+        # OVERRIDE
```

Comment:
we know that but the idea is to explain what's the purpose of the override

Reply:
explained

Reply:
no need to explain, this override is obvious...

Reply:
Yes, my comment was about all the `# OVERRIDE` everywhere, either we add a proper docstring explaining what the override does, either we don't write anything. But it is not needed to specify it is one, we can easily see this by the presence of `super()`